nntrainer/app_context.cpp

```diff
@@ -76,6 +76,7 @@
 #include <zoneout_lstmcell.h>
 
 #ifdef ENABLE_TFLITE_BACKBONE
+#include "upsample2d_layer.h"
```

Comment:
```suggestion
#include <upsample2d_layer.h>

#ifdef ENABLE_TFLITE_BACKBONE
```

Why did you include this header only under this condition?
It seems not to be matched with your `ac.registerFactory` condition as well.
It will cause some errors when `ENABLE_TFLITE_BACKBONE` option is disabled.

Reply:
:sweat: I made a mistake here. It was not intended. Thank you, @EunjuYang